src/lib.rs

```diff
@@ -170,6 +170,22 @@ extern crate bitflags;
 #[macro_use]
 extern crate objc;
 
+/// Taskbar Progress State
+pub enum TaskbarProgressState {
+  None,
+  Normal,
+  Intermediate,
+  Paused,
+}
+
+pub struct ProgressBarState {
+  pub state: Option<TaskbarProgressState>,
+  /// **Treated as Percentage**
+  pub progress: Option<f64>,
+  /// **Linux Only**
+  pub unity_uri: Option<String>,
+}
```

Comment:
```suggestion
/// A struct describing the state of the progress bar
pub struct ProgressBarState {
  /// The progress bar state
  pub state: Option<TaskbarProgressState>,
  /// The progress bar progress. This can be a value ranging from `0` to `100`
  pub progress: Option<f64>,
  /// An identifier for your app to communicate with the Unity desktop window manager **Linux Only**
  pub unity_uri: Option<String>,
}
```